web/package.json

```diff
@@ -190,6 +190,6 @@
   },
   "engines": {
```

Comment:
I tried with the newly supported `packageManager` field, but it requires a specific version and doesn't allow pinning just the major or minor version :(
```
  "packageManager": "pnpm@9.0.2",
```